packages/web-features-figuring/src/browser-compat-data/supportStatements.ts

```diff
@@ -168,13 +162,34 @@ export class RealSupportStatement extends SupportStatement {
       start = this.browser.version(this.version_added);
     }
 
+    let releases;
     if (this.version_removed === false) {
-      return this.browser.releases().filter((rel) => rel.compare(start) >= 0); // Release is on or after start
+      releases = this.browser.releases.filter((rel) => rel.compare(start) >= 0); // Release is on or after start
+    } else {
+      const end: Release = this.browser.version(this.version_removed);
+      releases = this.browser.releases.filter(
+        (rel) => rel.compare(start) >= 0 && rel.compare(end) < 0,
+      ); // Release is on after start and before the end
     }
 
-    const end: Release = this.browser.version(this.version_removed);
-    return this.browser
-      .releases()
-      .filter((rel) => rel.compare(start) >= 0 && rel.compare(end) < 0); // Release is on after start and before the end
+    let qualifications: Qualifications = {};
+    if (this.data.prefix) {
+      qualifications.prefix = this.data.prefix;
+    }
+    if (this.data.alternative_name) {
+      qualifications.alternative_name = this.data.alternative_name;
+    }
+    if (this.flags.length) {
+      qualifications.flags = this.flags;
+    }
+    if (this.partial_implementation) {
+      qualifications.partial_implementation = this.partial_implementation;
+    }
```

Comment:
This amounts to a filter of `this.data` for specific keys. But that's OK, it works.